src/nearcache/RepairingTask.ts

```diff
@@ -63,8 +76,8 @@ export class RepairingTask {
             return Promise.resolve(handler);
         }
 
-        const partitionService = this.client.getPartitionService() as PartitionServiceImpl;
-        handler = new RepairingHandler(objectName, partitionService, nearCache, this.localUuid);
+        const partitionService = this.partitionService as PartitionServiceImpl;
+        handler = new RepairingHandler(objectName, partitionService, nearCache, this.clientUuid);
```

Comment:
Maybe we can pass this directly to this constructor like 

```python
handler = new RepairingHandler(objectName, this.partitionService, nearCache, this.clientUuid);

```

Reply:
Sure, why not.